pills/07/simple_inherit.txt

```diff
@@ -0,0 +1,12 @@
+let
+  pkgs = import <nixpkgs> {};
+  inherit (pkgs) gcc coreutils;
+in
+  pkgs.stdenv.mkDerivation {
+    name = "simple";
+    builder = "${pkgs.bash}/bin/bash";
+    args = [ ./simple_builder.sh ];
+    inherit gcc coreutils;
```

Comment:
```suggestion
in
  pkgs.stdenv.mkDerivation {
    name = "simple";
    builder = "${pkgs.bash}/bin/bash";
    args = [ ./simple_builder.sh ];
    inherit (pkgs) gcc coreutils;
```

Otherwise that would be repeating `inherit` for no good reason